static/app/components/events/traceEventDataSection.tsx

```diff
@@ -375,6 +428,14 @@ export function TraceEventDataSection({
                 </SegmentedControl>
               </Tooltip>
             )}
+            <Tooltip title={t('Copy raw stacktrace to clipboard')}>
+              <Button
+                size="xs"
+                icon={<IconCopy />}
+                onClick={handleCopyRawStacktrace}
+                aria-label={t('Copy Raw Stacktrace')}
+              />
+            </Tooltip>
```

Comment:
use the built in button tooltip, tooltip without skipWrapper renders a span.

```suggestion
              <Button
                size="xs"
                icon={<IconCopy />}
                onClick={handleCopyRawStacktrace}
                aria-label={t('Copy Raw Stacktrace')}
                title={t('Copy raw stacktrace to clipboard')}
              />
```